streaming/streaming.go

```diff
@@ -722,7 +723,8 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 		fmt.Printf("flexTemplateRequest: %+v\n", req)
 		return internal.DataflowOutput{}, fmt.Errorf("unable to launch template: %v", err)
 	}
-	gcloudDfCmd := utils.GetGcloudDataflowCommand(req)
+	// Refactor to use accessor return value.
```

Comment:
@manitgupta when this is refactored to use accessors, the new LaunchFlexTemplateClient() will return jobid, gcloud command and error.